src/std_extensions/arithmetic/int_types.rs

```diff
@@ -244,6 +251,40 @@ mod test {
             ConstIntS::new(3, 128),
             Err(ConstTypeError::CustomCheckFail(_))
         );
-        assert_matches!(ConstIntS::new(3, -128), Ok(_));
+        assert!(ConstIntS::new(3, -128).is_ok());
```

Comment:
If the old version doesn't compile, fair, but if the assert fails, this new form will lose the reason why

Reply:
it does compile, it just adds an uncovered line 😅

Reply:
Oh of course, right. Remembered there was some issue with `assert_matches` but not what. Consider `unwrap()`...